flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/converters/SqlReplaceTableAsConverter.java

```diff
@@ -41,6 +41,7 @@
 /** A converter for {@link SqlReplaceTableAs}. */
 public class SqlReplaceTableAsConverter implements SqlNodeConverter<SqlReplaceTableAs> {
 
+    // TODO Update this converter if and when DISTRIBUTED BY is supported in REPLACE TABLE AS.
```

Comment:
remove TODO, the code base would be full of TODOs given the thousands of tickets we have in Flink